### PR TITLE
Volunteer manager

### PR DESCRIPTION
**Pull request by rathorevaibhav**
Changes include:

1. New role `volunteer-manager`
2. Changes in JobController and JobPolicy based on routes
3. Not checking edit and update method for authorization in JobController. Was giving an error. Will fix in another PR.

### CODE REVIEW

File: app/Policies/HR/JobPolicy.php [hunk 1]
```diff
@@ -19,7 +19,10 @@ class JobPolicy
      */
     public function view(User $user, Job $job)
     {
-        return $user->hasPermissionTo('hr_jobs.view');
+        if ($job->type == 'volunteer') {
```
Comment (pankaj-ag) [inline app/Policies/HR/JobPolicy.php]: Can we create a new function Isvolunteer().

File: app/Policies/HR/JobPolicy.php [hunk 2]
```diff
@@ -42,7 +48,10 @@ public function create(User $user)
      */
     public function update(User $user, Job $job)
     {
-        return $user->hasPermissionTo('hr_jobs.update');
+        if ($job->type == 'volunteer') {
```
Comment (pankaj-ag) [inline app/Policies/HR/JobPolicy.php]: Same as above. Make it a function.

File: app/Policies/HR/JobPolicy.php [hunk 3]
```diff
@@ -65,6 +77,9 @@ public function delete(User $user, Job $job)
      */
     public function list(User $user)
     {
-        return $user->hasPermissionTo('hr_jobs.view');
+        if (request()->is('hr/volunteers*')) {
```
Comment (pankaj-ag) [inline app/Policies/HR/JobPolicy.php]: This is fine but I think you can create a helper function also.

File: resources/views/hr/job/index.blade.php [hunk 4]
```diff
@@ -16,7 +16,11 @@
         @foreach ($jobs as $job)
         <tr>
             <td>
-                <a href="/hr/jobs/{{ $job->id }}/edit">{{ $job->title }}</a>
+                @if ($type == 'volunteer')
```
Comment (pankaj-ag) [inline resources/views/hr/job/index.blade.php]: Function?

**Comment by rathorevaibhav** [comment]
@pankaj-ag thanks. Since this is a priority item, I'm merging it. Targeting feedback in the next one.

**Pull request merged**